src/twisted/internet/error.py

```diff
@@ -236,6 +236,17 @@ class ConnectionAborted(ConnectionLost):
     @since: 11.1
     """
 
+    def __str__(self):
+        s = [(
+            "Connection was aborted locally using"
+            " ITCPTransport.abortConnection"
+        )]
+        if self.args:
+            s.append(': ')
+            s.append(' '.join(self.args))
```

Comment:
This is potentially an incompatible change; `args` will now raise an exception if you put anything but native `str` into it.